File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L5/05/method.kt
package IR

import java.util.*

object method {
    // method reverse
    fun beautyReverse(num: Int) {
        var n = num
        while (n != 0) {
            val r = n % 10
            print(r)
            n /= 10
        }
        println()
    }

    fun reverse(n: Int) {
        beautyReverse(n)
    }

    // main program
    @JvmStatic
    fun main(args: Array<String>) {
        print("Enter an integer: ")
        val input = Scanner(System.`in`)
        val n = input.nextInt()
        // call method
        reverse(n)
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L3/05/method.ts
import { Scanner } from 'readline-sync';

class Method {
  // method reverse
  public static beautyReverse(num: number): void {
    while (num !== 0) {
      const r = num % 10;
      console.log(r);
      num = Math.floor(num / 10);
    }
    console.log();
  }

  // main program
  public static main(args: string[]): void {
    console.log('Enter an integer: ');
    const input = new Scanner();
    const n = input.nextInt();
    // call method
    Method.beautyReverse(n);
  }
}

